src/app/modules/users/components/users-list/users-list.component.spec.ts

```diff
@@ -59,6 +69,21 @@ describe('UsersListComponent', () => {
     expect(component.users).toEqual(state.data);
   });
 
+  it('on success load users, the data of role should be an array and role null', () => {
```

Comment:
Maybe you could add a new test with the inverse case of role and roles... so you can modify the `data` on line 17 to have info in `role`, and `roles` as empty

Reply:
The inverse case should have `role` as string and `roles` as null.

Reply:
Ok, I have tested that case also.